build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs

```diff
@@ -15,6 +15,16 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class AzurePublisher
     {
+        public enum Products
+        {
+            SharedFramework,
+            Host,
+            SDK,
+            HostAndFramework,
+            FrameworkAndSdk,
+            CombinedPackage,
```

Comment:
Package is somewhat ambiguous given the 4 or 5 different kinds of packages we ship out of the CLI.

What do you think of `CombinedHostFrameworkSdk` to make it more explicit?